### PR TITLE
fix: azure monitoring enable for azure red hat openshift clusters

### PR DESCRIPTION
**Pull request by ganga1980**
This PR has the fix for enabling azure monitor for containers for azure red hat openshift clusters.
Fix is to use latest API version 2019-09-30-preview since this version has required API changes for monitoring integration for ARO clusters.
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] The PR has modified HISTORY.rst describing any customer-facing, functional changes. Note that this does not include changes only to help content. (see [Modifying change log](https://github.com/Azure/azure-cli/tree/master/doc/authoring_command_modules#modify-change-log)).

- [x] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

### CODE REVIEW

File: src/azure-cli/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -3208,26 +3207,15 @@ def openshift_create(cmd, client, resource_group_name, name,  # pylint: disable=
...
-            master_pool_profile=agent_master_pool_profile,
-            router_profiles=[default_router_profile],
-            monitor_profile=monitor_profile)
-    else:
-        osamc = OpenShiftManagedCluster(
-            location=location, tags=tags,
-            open_shift_version="v3.11",
-            network_profile=network_profile,
-            auth_profile=auth_profile,
-            agent_pool_profiles=agent_pool_profiles,
```
Comment (amanohar) [inline src/azure-cli/azure/cli/command_modules/acs/custom.py]: Why is `else` case removed?

**Comment by yonzhan** [comment]
add to S162.

File: src/azure-cli/azure/cli/command_modules/acs/_client_factory.py [hunk 2]
```diff
@@ -64,7 +64,7 @@ def get_container_service_client(cli_ctx, **_):
 def get_osa_container_service_client(cli_ctx, **_):
     from azure.mgmt.containerservice import ContainerServiceClient
 
-    return get_mgmt_service_client(cli_ctx, ContainerServiceClient, api_version='2019-04-30')
+    return get_mgmt_service_client(cli_ctx, ContainerServiceClient, api_version='2019-09-30-preview')
```
Comment (ehashman) [inline src/azure-cli/azure/cli/command_modules/acs/_client_factory.py]: Is this what we want? I was under the impression that we don't want to use the new API version for the CLI unless customers are using the new features (e.g. Azure Monitor). cc @amanohar

Comment (ehashman) [reply src/azure-cli/azure/cli/command_modules/acs/_client_factory.py]: As discussed on our Azure Monitor meeting today, we are okay with moving all the AZ CLI functionality to the new preview API, as multiple versions are not supported.

**Pull request merged**